martin/src/srv/fonts.rs

```diff
@@ -18,7 +19,8 @@ struct FontRequest {
 #[route(
     "/font/{fontstack}/{start}-{end}",
     method = "GET",
-    wrap = "middleware::Compress::default()"
+    wrap = "middleware::Compress::default()",
+    wrap = "Etag"
```

Comment:
a bit weird here - the above wrap is using `::default()`, but this one does not?  Would it make sense to add both `Compress` and `Etag` to the `use` statements at the top, and have a compact `wrap = "Compress", wrap = "Etag"` ?

Reply:
And what's the order here?  
```rust
let app = App::new()
    .wrap(MiddlewareA::default())
    .wrap(MiddlewareB::default())
    .wrap(MiddlewareC::default())
    .service(service);
```
The request first gets processed by the middleware specified last - MiddlewareC.

The request would be processed by Compress first or Etag here? 

Note: The Etag must be loaded as the last middleware.

Reply:
A second thought: Compare the etag before compressing seems Ok but seems a little bit tricky?

Reply:
No clue how this is handled on the way out of the request.
Correctly, ertags should be checked first, then compression as that enables the fast-path there.

Will have to benchmark which one is faster

Reply:
> above wrap is using ::default(), but this one does not

Yes, clippy complained that I am using default on a unit struct.

Reply:
Have resorted the wraps as suggested.
Benchmarking shows a barely measurable improvement. Compressing so small thigns is inherently fast.

For future reference:

![image](https://github.com/user-attachments/assets/5f7a8518-5eba-48f9-bf1e-c2ba6c879c60)